mail/imapdedup/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        quentinsf IMAPdedup ec56dbdc687c73808336fc38098c268a0177e57a
+
```

Comment:
This looks good. Let's add a `revision` field and it should be good to go:
```
github.setup        quentinsf IMAPdedup ec...
revision            0
```

Reply:
it's done!